frontend/src/screens/learner.tsx

```diff
@@ -145,34 +175,38 @@ export const MobileStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies })
                     slideShadows: false,
                     perSlideOffset: 14,
                 }}
-                centeredSlides={true}
-                pagination={{
-                    enabled: true,
-                    dynamicBullets: true,
-                    dynamicMainBullets: 5,
-                }}
-                modules={[EffectCards, Pagination]}
-                style={{
-                    paddingBottom: '2rem',
-                    marginBottom: '1rem',
-                }}
+                centeredSlides
+                modules={[EffectCards]}
+                style={{ paddingBottom: '2rem', marginBottom: '1rem' }}
+                pagination={false}
             >
-                {studies.map((study) => (
-                    <SwiperSlide key={study.id} className="pb-1">
-                        <StudyCard study={study} />
-                    </SwiperSlide>
-                ))}
+                {studies
+                    .sort((a, b) =>
```

Comment:
Maybe this can be extracted out to a function with a good name? its a little hard to read